echo-webhooks/src/test/java/com/netflix/spinnaker/echo/scm/BitbucketWebhookEventHandlerTest.java

```diff
@@ -51,6 +52,7 @@ void canHandlePayload() throws IOException {
     event.content.put("event_type", "repo:push");
 
     BitbucketWebhookEventHandler handler = new BitbucketWebhookEventHandler();
-    assertThatCode(() -> handler.handle(event, payload)).doesNotThrowAnyException();
```

Comment:
Any way we could get a slightly more rigorous test for this?

Reply:
Done 👍